drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -280,6 +280,18 @@ mlx5_vdpa_pci_remove(struct rte_pci_device *pci_dev __rte_unused)
 }
 
 static const struct rte_pci_id mlx5_vdpa_pci_id_map[] = {
+	{
```

Comment:
how about having all the devices exists on the net PMD? not only VFs.

Reply:
Thanks @shahafsh 
Do u mean add here all the devices (and not only VFs)?
the net PMD exits on all devices with vdpa=1 devarg ...

Reply:
yes, since it is for dev purpose, let's have it all.